--- elasticsearch-6.0.0-beta2/bin/elasticsearch-env	2017-08-04 01:12:25.000000000 +0200
+++ elasticsearch-6.0.0-beta2/bin/elasticsearch-env	2017-08-18 18:26:53.721617370 +0200
@@ -20,7 +20,10 @@
 # determine Elasticsearch home; to do this, we strip from the path until we find
 # bin, and then strip bin (there is an assumption here that there is no nested
 # directory under bin also named bin)
-ES_HOME=`dirname "$SCRIPT"`
+if [ -z "$ES_HOME" ]; then
+    echo "You must set the ES_HOME var" >&2
+    exit 1
+fi
 
 # now make ES_HOME absolute
 ES_HOME=`cd "$ES_HOME"; pwd`
