--- elasticsearch-8.8.0/bin/elasticsearch-env   1970-00-00 00:00:00.000000001 +0000
+++ elasticsearch-8.8.0/bin/elasticsearch-env   1970-00-00 00:00:00.000000001 +0000
@@ -19,19 +19,16 @@
   fi
 done
 
-# determine Elasticsearch home; to do this, we strip from the path until we find
-# bin, and then strip bin (there is an assumption here that there is no nested
-# directory under bin also named bin)
-ES_HOME=`dirname "$SCRIPT"`
+# determine Elasticsearch home
+# expect to be set to the directory containing "bin"
+if [ -z "$ES_HOME" ]; then
+    echo "You must set the ES_HOME var" >&2
+    exit 1
+fi
 
 # now make ES_HOME absolute
 ES_HOME=`cd "$ES_HOME"; pwd`
 
-while [ "`basename "$ES_HOME"`" != "bin" ]; do
-  ES_HOME=`dirname "$ES_HOME"`
-done
-ES_HOME=`dirname "$ES_HOME"`
-
 # now set the path to java
 if [ ! -z "$ES_JAVA_HOME" ]; then
   JAVA="$ES_JAVA_HOME/bin/java"
