--- elasticsearch-5.4.1/bin/elasticsearch	1970-00-00 00:00:00.000000001 +0000
+++ elasticsearch-5.4.1/bin/elasticsearch	1970-00-00 00:00:00.000000001 +0000
@@ -105,7 +105,10 @@
 done
 
 # determine elasticsearch home
-ES_HOME=`dirname "$SCRIPT"`/..
+if [ -z "$ES_HOME" ]; then
+    echo "You must set the ES_HOME var" >&2
+    exit 1
+fi
 
 # make ELASTICSEARCH_HOME absolute
 ES_HOME=`cd "$ES_HOME"; pwd`
--- elasticsearch-5.4.1/bin/elasticsearch-plugin	1970-00-00 00:00:00.000000001 +0000
+++ elasticsearch-5.4.1/bin/elasticsearch-plugin	1970-00-00 00:00:00.000000001 +0000
@@ -16,7 +16,10 @@
 done
 
 # determine elasticsearch home
-ES_HOME=`dirname "$SCRIPT"`/..
+if [ -z "$ES_HOME" ]; then
+    echo "You must set the ES_HOME var" >&2
+    exit 1
+fi
 
 # make ELASTICSEARCH_HOME absolute
 ES_HOME=`cd "$ES_HOME"; pwd`
